pkg/roachprod/cloud/gc.go

```diff
@@ -442,6 +443,8 @@ func GCClusters(l *logger.Logger, cloud *Cloud, dryrun bool) error {
 
 	var destroyedClusters []resourceDescription
 	for _, c := range s.destroy {
+		_ = promhelperclient.NewPromClient().DeleteClusterConfig(context.Background(),
```

Comment:
See comment about calling `DeleteClusterConfig` in `cloud.DestroyCluster`, but if for some reason we can't do that, we should make this compatible with `dryRun`.